### PR TITLE
Update bot creation docs

### PR DESCRIPTION
**Pull request by rahul1**

### CODE REVIEW

File: packages/docs/docs/app/bots.md [hunk 1]
```diff
@@ -28,25 +25,120 @@ By using Bots, the entire logic is self contained and managed in one place. Like
...
+You can see all Bots in your account on the Bot resource page: [https://app.medplum.com/Bot](https://app.medplum.com/Bot)
+
+Click on your new Bot and navigate to the **Editor** tab. This presents a code editing window where you can write your
+Javascript code.
+
+![Bot Editor](/img/app/bots/BotResourcePage.gif)
+
+All Bots are simply a single Javascript file with a standard async entry point function called `handler`. To start your bot, create a `handler` function as shown below in the **Editor** window.
 
 ```javascript
```
Comment (codyebberson) [inline packages/docs/docs/app/bots.md:67]: Maybe use typescript here?

```typescript
import { BotEvent, MedplumClient} from '@medplum/core';

export async function handler(medplum: MedplumClient, event: BotEvent): Promise<any> {
  // Your code here
}
```

**Review by codyebberson — APPROVED** [review]
👍

File: packages/docs/docs/app/bots.md (hunk 1 above)
Comment (rahul1) [reply packages/docs/docs/app/bots.md:70]: Trying to keep it in sync with the in-browser Editor window for right now which needs to be javascript (matches the rest of the tutorial)

File: packages/docs/docs/app/bots.md [hunk 2]
```diff
@@ -28,100 +28,191 @@ By using Bots, the entire logic is self contained and managed in one place. Like
...
 
-Example:
+#### `ACCESS_TOKEN`
 
-```javascript
-export async function handler(medplum, event) {
-  console.log('Hello world');
-}
-```
+This is the `access_token` you receive after completing the OAth authentication flow. See [this tutorial](/tutorials/api-basics/create-fhir-data#authenticating-using-oauth-client-credentials-flow) for more information.
```
Comment (reshmakh) [inline packages/docs/docs/app/bots.md:153]: Typo OAuty

**Review by reshmakh — APPROVED** [review]

**Pull request merged**